File: beginner/java/1005.java
import java.io.IOException;
import java.util.Locale;
import java.util.Scanner;

public class Main {
    public static void main(String[] args) throws IOException {
        Scanner scanner = new Scanner(System.in);

            scanner.useLocale(Locale.US);

            double nota1 = scanner.nextDouble();
            double nota2 = scanner.nextDouble();

            double media = ((nota1 * 3.5) + (nota2 * 7.5)) / 11;

            System.out.printf("MEDIA = %.5f%n", media);


        scanner.close();
    }
}

File: beginner/c/1005.c
#include <stdio.h>

int main(){

    double nota1, nota2;

    scanf("%lf", &nota1);
    scanf("%lf", &nota2);

    double media = ((nota1 * 3.5) + (nota2 * 7.5)) / 11;

    printf("MEDIA = %.5lf\n", media);

    return 0;
}
